test-assets/7677a9/slider.js

```diff
@@ -0,0 +1,44 @@
+function increaseSlider() {
+    document.getElementById('motionSlider').value++;
+    document.getElementById('output').innerHTML = document.getElementById('motionSlider').value;
+}
+
+function decreaseSlider() {
+    document.getElementById('motionSlider').value--;
+    document.getElementById('output').innerHTML = document.getElementById('motionSlider').value;
+}
+
+function handleOrientation(event) {
+    if (event.gamma > 20) {
+        increaseSlider();
+    } else if (event.gamma < -20) {
+        decreaseSlider();
+    }
+}
```

Comment:
I'd also factor in something that take `gamma` (rather than `event` as parameter) and have the other function only set gamma to a correct value (either extracting it from `even` or looking at `disableMotion.checked` on top of it).
Again, not really necessarily.